app/views/general_shopping_lists/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<p style="color: green"><%= notice %></p>
+
+<div id="recipes">
+  <h1>Shopping List</h1>
+  <div class="topBoxRecipeList">
+    <div class="leftSideRecipeList">
+      <p>Amount of food items to buy: <%= @missing_foods_price %> </p>
+    </div>
+
+    <div class="rightSideRecipeList">
+      <p>Total value of food needed: <%= @missing_foods_count %> </p>
+    </div>
+  </div>
+  
+  <hr class="horizonatalLine">
+  <table class="tableRecipeList">
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Price</th>
```

Comment:
- [ ] Let's kindly make sure that the **Price** column is in ($) as seen in the wireframe